tools/code_format/config.yaml

```diff
@@ -215,14 +218,22 @@ paths:
     - test/extensions/stats_sinks/wasm/wasm_stat_sink_test.cc
     - test/test_common/wasm_base.h
 
-dir_order:
-  - envoy
-  - common
-  - source
-  - exe
-  - server
-  - extensions
-  - test
+dir_order:                                                                     # unique
+  - envoy                                                                      # unique
+  - nighthawk                                                                  # unique
+  - external/source/envoy                                                      # unique
+  - external                                                                   # unique
+  - api                                                                        # unique
+  - common                                                                     # unique
+  - source                                                                     # unique
+  - exeserver                                                                  # unique
```

Comment:
Should these be two entries `exe` and `server`?

Reply:
Yes, fixed.